src/components/WhiteContainer/index.tsx

```diff
@@ -5,9 +5,14 @@ import theme from "tools/theme";
 type WhiteContainerProps = {
   className?: string;
   children?: ReactNode;
+  onClick?: () => void;
 };
```

Comment:
```suggestion
type WhiteContainerProps = {
  children?: ReactNode;
}  & HTMLProps<HTMLDivElement>;;
```

이러한 구조로 `type` 변경하면 `onClick` 외에도 다른 `div`의 props 전달받을 수 있을 것 같아요!

Reply:
수정했습니다!